app/scripts/controllers/metametrics.js

```diff
@@ -569,7 +569,6 @@ export default class MetaMetricsController {
       ).length,
       [TRAITS.NUMBER_OF_TOKENS]: this._getNumberOfTokens(metamaskState),
       [TRAITS.OPENSEA_API_ENABLED]: metamaskState.openSeaEnabled,
-      [TRAITS.THREE_BOX_ENABLED]: metamaskState.threeBoxSyncingAllowed,
```

Comment:
Could we, instead, set this to false and put a comment that it was set to false due to 3box removal? In a future PR we can remove it entirely but i'd like to see the user trait change gradually..If we delete this it'll be stuck in 'true' mode for those users who had it enabled before this is rolled out

Reply:
Good idea! Done here: 46ab5a404450c4ec7b893afb5aa8fb6af42109cd